ts/private/ts_project.bzl

```diff
@@ -84,6 +86,15 @@ def _ts_project_impl(ctx):
 
     srcs = [_lib.relative_to_package(src.path, ctx) for src in srcs_inputs]
 
+    srcs_deps = ctx.attr.srcs + ctx.attr.deps
+
+    tsc_inputs = copy_files_to_bin_actions(ctx, srcs_inputs + tsconfig_inputs)
```

Comment:
I think we can probably still improve the readability of this a bit. Let me know if you have any new ideas/names etc.